src/test/components/Agent/AgentInfo.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react'
+import AgentInfo from 'src/components/AgentInfo'
+import Agent from 'src/types/Agent'
+import Office from 'src/types/Office'
+
+const anytownOffice: Office = {
+  id: 0,
+  street: '123 Noendinsight Circle',
+  city: 'Anytown',
+  state: 'CA',
+  zip: '12345',
+  type: 'Office',
+}
+const agentWithFirstName: Agent = {
+  firstName: 'Jebediah E.',
+  lastName: 'Hoover',
+  fullName: 'Jebediah E. Hoover',
+  office: anytownOffice,
+  id: '54321',
+  averageRating: 1,
+  type: 'Agent',
+}
+
+const agentWithoutFirstName: Agent = {
+  lastName: 'Hoover',
+  fullName: 'Hoover',
+  office: anytownOffice,
+  id: '54321',
+  averageRating: 1,
+  type: 'Agent',
+}
```

Comment:
[Non-Blocking Suggestion]: Making modified copies of existing mock data can lighten up test code and make it clearer what's changing from one to the other.
```suggestion
const agentWithoutFirstName = { ...agentWithFirstName, firstName: undefined }
```